content_scripts/link_hints.js

```diff
@@ -419,6 +419,7 @@ class LinkHintsMode {
         top: 0,
         left: 0,
         position: "absolute",
+        display: 'block',
```

Comment:
Without the GHE styling interfering, this element otherwise has a computed display of `block`